core/iwasm/compilation/aot_emit_function.c

```diff
@@ -3133,7 +3177,7 @@ aot_compile_op_call_ref(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
         PUSH(result_phis[i], func_type->types[func_param_count + i]);
     }
 
-    if (comp_ctx->enable_aux_stack_frame) {
+    if (comp_ctx->aux_stack_frame_type == AOT_STACK_FRAME_TYPE_STANDARD) {
```

Comment:
Should be `if (comp_ctx->aux_stack_frame_type
        && !comp_ctx->call_stack_features.frame_per_function) {` like L2982?